src/events.ts

```diff
@@ -35,6 +35,18 @@ type InvestmentAccountProvisionRequestedMessage = Message & {
   errorUrl: string;
 }
 
+type Money = {
+  quantity: number;
+  currency: string;
+}
+
+type InvestmentSellRequestedMessage = Message & {
+  amount: Money;
+  callbackPath: string;
```

Comment:
one thing (I'm guilty of that) is that we have a clear `cancelUrl` `errorUrl` and then a weird `callbackPath` instead of e.g. `successUrl` (or rather, `successPath` it's not a full url right?) but then you'd have to change it in the account provisioning flow as well....

Reply:
can we rename this to callbackUrl?

Reply:
In this context/PR for sure we can rename it - is it feasible to also change it for accountprovisioning or do we keep that separate?

Reply:
we can keep the changes to the context of the PR adn then refactor the other thing imo